libzecale/tests/circuits/bls12_377_pairing_test.cpp

```diff
@@ -0,0 +1,381 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+#include "libzecale/circuits/fields/fp12_2over3over2_gadgets.hpp"
+#include "libzecale/circuits/pairing/bls12_377_pairing.hpp"
+#include "libzecale/circuits/pairing/bw6_761_pairing_params.hpp"
+
+#include <gtest/gtest.h>
+#include <libff/algebra/curves/bls12_377/bls12_377_pp.hpp>
+#include <libff/algebra/curves/bw6_761/bw6_761_pp.hpp>
+#include <libsnark/gadgetlib1/gadgets/pairing/pairing_params.hpp>
+#include <libzeth/snarks/groth16/groth16_snark.hpp>
+
+using ppp = libff::bw6_761_pp; // Parent pairing
+using ps = libsnark::pairing_selector<ppp>;
+using cpp = ps::other_curve_type; // Child pairing
```

Comment:
I would stick to the notions of "wrapping" and "nested" instead of "parent" and "child"

Reply:
See: https://github.com/clearmatics/zecale#notes

Reply:
Switched to wpp and npp.

Reply:
What about modifying the comments consistently?